app/models/ticket_subscription.rb

```diff
@@ -2,6 +2,15 @@ class TicketSubscription < ApplicationRecord
   belongs_to :ticket
   belongs_to :user
 
+  validates :ticket_id,
+    presence: true
+
+  validates :user_id,
+    presence: true,
+    uniqueness: {
```

Comment:
I do not need it now.
If you can purchase multiple tickets and you can modify them, you probably need to record your purchase history.
It is supposed to answer purchase inquiries from users.